app/controllers/pieces_controller.rb

```diff
@@ -0,0 +1,14 @@
+class PiecesController < ApplicationController
+	def show 
+		@piece = Piece.find(params[:id])
+	end
+
+	def update
+		@piece = Piece.find(params[:id])
+		@game = Game.find(params[:id])
+		if @piece.valid_move?([:x, :y]) && @game(piece_in_space?(x, y) == false)
+			@piece.update_attributes(position_x: x, position_y: y)
+		end	
+		redirect_to game_path(@game)
+	end
+end
```

Comment:
We have a controller but there is no piece view to go to. We need to create a view for pieces that displays the board. There's quite a bit of missing logic here that we will have to go over.